Sources/CellParentViewProtocol.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  PlanGrid
+//  https://www.plangrid.com
+//  https://medium.com/plangrid-technology
+//
+//  Documentation
+//  https://plangrid.github.io/ReactiveLists
+//
+//  GitHub
+//  https://github.com/plangrid/ReactiveLists
+//
+//  License
+//  Copyright © 2018-present PlanGrid, Inc.
+//  Released under an MIT license: https://opensource.org/licenses/MIT
+// 
+
+import Foundation
+import UIKit
+
+/**
+ This protocol unifies `UICollectionView` and `UITableView` by providing a common dequeue method for cells.
+ It describes a view that is the "parent" view for a cell.
+ For `UICollectionViewCell`, this would be `UICollectionView`.
+ For `UITableViewCell`, this would be `UITableView`.
+ */
+protocol CellParentViewProtocol {
```

Comment:
How do you feel about `Container` versus `Parent`?

Reply:
🤷‍♂️ 

either works for me

Reply:
Also like container :)

Reply:
will update!